### PR TITLE
Made changes required to serve google_calendar.gif from indico

### PR DESCRIPTION
**Pull request by pret3nti0u5**
In reference to issue #4660.
Also, since this was a minor change, as advised this PR request has been submitted without testing. Please add the `hacktoberfest-accepted` label if you're happy with the changes :)

### CODE REVIEW

File: indico/modules/events/templates/footer.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h4>{% trans %}Calendaring{% endtrans %}</h4>
             </div>
             <div class="social-site">
                 <a href="https://www.google.com/calendar/event?{{ google_calendar_params|urlencode }}" target="_blank">
-                    <img src="//www.google.com/calendar/images/ext/gc_button1.gif" alt="" border="0">
+                    <img src="{{ indico_config.IMAGES_BASE_URL }}/google_calendar.gif" alt="" border="0">
```
Comment (ThiefMaster) [inline indico/modules/events/templates/footer.html:48]: Please use `url_for` using the `assets.image` endpoint. You find examples if you search for that endpoint name in existing templates.

File: indico/modules/events/templates/footer.html [hunk 2]
```diff
@@ -45,7 +45,7 @@ <h4>{% trans %}Calendaring{% endtrans %}</h4>
             </div>
             <div class="social-site">
                 <a href="https://www.google.com/calendar/event?{{ google_calendar_params|urlencode }}" target="_blank">
-                    <img src="//www.google.com/calendar/images/ext/gc_button1.gif" alt="" border="0">
+                    <img src="{{ url_for('assets.image', filename="google_calendar.gif") }}" alt="" border="0">
```
Comment (ThiefMaster) [inline indico/modules/events/templates/footer.html:48]:
```suggestion
                    <img src="{{ url_for('assets.image', filename='google_calendar.gif') }}" alt="" border="0">
```

**Review by ThiefMaster — APPROVED** [review]

**Comment by ThiefMaster** [comment]
Thanks, looks good now!

**Comment by ThiefMaster** [comment]
Since your PR is merged, it should count for the event even without the label as the repo itself is opted in.

**Comment by pret3nti0u5** [comment]
> 
> 
> Since your PR is merged, it should count for the event even without the label as the repo itself is opted in.

Got it. Thanks for being so patient with me. I'll see if I can contribute more to this project even after the end of Hacktober Fest.

**Pull request merged**